pages/pipelines/command_step.md.erb

```diff
@@ -454,10 +454,7 @@ steps:
 
 ## Fail fast
 
-To automatically cancel remaining jobs as soon the first job fails (except jobs that you've marked as `soft_fail`):
-
-1. Make sure that _Publish failed state early_ is checked in your _Pipeline Repository_ settings
-2. Mark jobs that you want to cancel immediately with `cancel_on_build_failing: true`
+To automatically cancel remaining jobs as soon the first job fails (except jobs that you've marked as `soft_fail`), mark jobs that you want to cancel immediately with `cancel_on_build_failing: true`.
```

Comment:
I've added a couple of missing words to the first part and used some of the explanation in the changelog entry for the second part:

```suggestion
To automatically cancel any remaining jobs as soon as the first job fails (except jobs that you've marked as `soft_fail`), add the `cancel_on_build_failing: true` attribute to your command steps.
```

I think avoiding some of the repetition makes it easier to read. What do you think?

Reply:
awesome! thanks @mbelton-buildkite